packages/model-typings/src/models/IUsersModel.ts

```diff
@@ -39,11 +39,19 @@ export interface IUsersModel extends IBaseModel<IUser> {
 	findPaginatedByActiveUsersExcept<T = IUser>(
 		searchTerm: any,
 		exceptions: any,
-		options: any,
 		searchFields: any,
+		options?: any,
 		extraQuery?: any,
 		params?: { startsWith?: boolean; endsWith?: boolean },
 	): FindPaginated<FindCursor<T>>;
+	findPaginatedActiveUsersByRoomIdWithHighestRole<T = IUser>(
+		searchTerm: any,
```

Comment:
Done! Since the params for this method are similar to the ones in `findPaginatedByActiveUsersExcept`, I was also trying to type it, but that was harder than I thought :zipper_mouth_face: at least the new one is typed now